game/views.py

```
@@ -1,1157 +1,1172 @@
+""",
+                    from_email=None,
+                    recipient_list=[user.email],
+                    fail_silently=False,
+                )
+
+                messages.success(
+                    request,
+                    'Confirmation email sent to your registered email.'
+                )
+
+            except Exception:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)